File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A2/027/027.cpp
#include<bits/stdc++.h>
using namespace std;
int main(){
    int n;
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }
    int max_score = -1e9;
    for(int i=0;i<n;i++){
        max_score = max(max_score,arr[i]);
    }
    int count = 0;
    for(int i=0;i<n;i++){
        if(arr[i] == max_score){
            count++;
        }
    }
    cout<<max_score<<endl<<count<<endl;
    return 0;
}

File: Resources/Projects/Archive/2023/Competition/Programming/Contest/TOI-Zero_68/A2/019/TOI-Zero_68/A2/027/027.py
n = int(input())
arr = []
for _ in range(n):
    arr.append(int(input()))

max_score = -1000000000
for i in range(n):
    max_score = max(max_score, arr[i])

count = 0
for i in range(n):
    if arr[i] == max_score:
        count += 1

print(max_score)
print(count)